app/views/amendments/browse_section.html.erb

```diff
@@ -1,27 +1,32 @@
 <div>
   <% if section %>
-    <%= browse_title(section, browse_section_amendment_path) %>
+    <% if params['locked_section'] == 'false' %>
```

Comment:
try not to access params directly from the view, is better to add a `locked_section?` helper method in the controller